src/components/Editor.jsx

```diff
@@ -1,6 +1,6 @@
 import React, { PropTypes } from 'react';
 import CodeMirror from 'react-codemirror';
-import _ from 'lodash';
+import isEqual from 'lodash/isEqual';
```

Comment:
Yeah I get that but you're choosing to spare the memory and sacrifice speed this way